pyproject.toml

```diff
@@ -139,23 +139,27 @@ skip_gitignore = true
 
 # https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml
 [tool.mypy]
-mypy_path = "src/package/:tests/"
```

Comment:
When we run `mypy` via `make` or `pre-commit` then the files are gathered here https://github.com/jenstroeger/python-package-template/blob/6917fcc612d4cd059c5886588182b4cf86fd01ab/.pre-commit-config.yaml#L87-L88 and passed to `mypy`. So I commented out this setting because it duplicates where files are collected. I thought it’s better to gather files in one place only — in this case via `pre-commit`.